nameko_sqlalchemy.py

```diff
@@ -22,11 +22,14 @@ def setup(self):
             'service_name': service_name,
             'declarative_base_name': decl_base_name,
         })
+        self.engine = create_engine(self.db_uri)
+
+    def stop(self):
+        self.engine.dispose()
```

Comment:
I see. The only doubt I have is that it seems there is no difference in the testing between the old behaviour and the new one. If the old one is causing issues, maybe it could be worth to come up with a very simple test that would break if someone would revert to the old logic in the future?